std/log/levels.ts

```diff
@@ -1,26 +1,51 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
-export const LogLevel: Record<string, number> = {
-  NOTSET: 0,
-  DEBUG: 10,
-  INFO: 20,
-  WARNING: 30,
-  ERROR: 40,
-  CRITICAL: 50,
-};
+export enum LogLevels {
```

Comment:
`LogLevels` and `LogLevel` are too similar to each other, I wasn't sure what was the change for a few minutes. Could we use some more distinct names? Also, it'd be great if you could add JSdocs explaining purpose of each export

Reply:
JSDoc--sure, no problem. I found the nearness of the symbols to be a _strength_--that proximity infers their tight coupling and consolidates your import lookup keystrokes. With that said, if you still feel strongly about it, i'm happy to take a prescription and just type them in. `LogLevel` to just `Level` or `LevelName`, perhaps?

Reply:
`LevelName` seems to bode well, as it also has parity with `levelName` args in various existing functions. tell me how you like e6d508b